src/main/java/com/ridge/novi/techiteasy/models/Television.java

```diff
@@ -0,0 +1,186 @@
+package com.ridge.novi.techiteasy.models;
+
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class Television {
+    public static Television existing;
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    Long id;
```

Comment:
Het is wel goed om hier private voor alle velden te zetten. Nu zijn ze package private, dus alleen in de models package kan je bij deze velden zonder getter/setter maar ook dat zou je niet willen